packages/marshal/src/marshal.js

```diff
@@ -247,52 +247,57 @@ function isPassByCopyArray(val) {
   return true;
 }
 
+/**
+ * Everything having to do with a `dataProto` is a temporary kludge until
+ * we're on the other side of #2018. TODO remove.
+ */
+const dataProto = harden(
+  create(objectPrototype, {
+    [PASS_STYLE]: { value: 'copyRecord' },
```

Comment:
just like `REMOTE_STYLE` below, this magic string should be in a constant.

Reply:
`REMOTE_STYLE` is an outlier because we know the name it has (`'presence'`) is not the name we want (`'remote'`). For the other's, the string literal style is pervasive in our code. We should discuss sometime the tradeoffs. There are a lot of places in JS where a misspelling can lead to something silently going astray, such as property or method dispatch. Our properjs typescript/jsdoc types help a lot with these. They also do OR typing of an enumerated set of string literals. So I don't know that the string-literal-as-enum is really a different hazard than property or method names. 

We should definitely examine this question. But I'm not gonna take on a pervasive style change in this narrowly targeted PR.